<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistDisplayWidget</class>
 <widget class="QWidget" name="PlaylistDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>258</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonClear">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset resource="../../../data/icons.qrc">
       <normaloff>:/icons/24x24/clear.png</normaloff>:/icons/24x24/clear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="comboCategory">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSearch">
     <property name="text">
      <string>Search:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelCategory">
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelLanguage">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="comboLanguage">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QListView" name="playlistView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>editSearch</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
